src/components/views/dialogs/DevtoolsDialog.tsx

```diff
@@ -14,8 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React, {useState, useEffect} from 'react';
-import PropTypes from 'prop-types';
+import React, {useState, useEffect, ChangeEvent, MouseEvent} from 'react';
```

Comment:
spaaaaaccesss

Reply:
I went through and updated all imports for this file, hopefully that's okay...